XYZDrivers/src/main/java/com/xyzdrivers/models/Claim.java

```diff
@@ -42,7 +42,7 @@ public String getStatus() {
         return this.status;
     }
 
-    public float getAmount() {
+    public double getAmount() {
```

Comment:
Why was this changed from float to double?

Reply:
It's a DOUBLE in the SQL db, so converting it to a float was throwing an error.

Reply:
retrieving it as a float was throwing an error*, not converting.
Regardless, I figured it would be better to change the type than convert since it's a double in the DB.

Reply:
Ah right, I thought it was a float for some reason. Yeah change to double was good shout